app/qml/components/PasswordField.qml

```diff
@@ -0,0 +1,95 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.0
+import QtQuick.Layouts 1.3
+import QtGraphicalEffects 1.0
+import lc 1.0
+import "../" // import InputStyle singleton
+
+Row {
+  id: root
+  spacing: 0
+
+  property var fontColor
+  property var bgColor
+  property var password: password
+  property string placeholder: 'Password'
```

Comment:
can we rename the property to `placeholderText` to make it consistent with other items like TextField from Qt